deploy/rbac.yaml

```diff
@@ -19,9 +19,9 @@ metadata:
   labels:
     name: litmus
 rules:
-- apiGroups: ["*"]
-  resources: ["*"]
-  verbs: ["*"]
+- apiGroups: ["","apps","batch","litmuschaos.io"]
+  resources: ["pods","jobs","deployments","daemonsets","events","configmaps","services","chaosengines","chaosexperiments"]
```

Comment:
is there any need to add finalizer resources separately ?

Reply:
Yes, It is showing error If we are not passing these resources and  deletecollection verb

Reply:
though we aren't using in today, can you also add chaosResults

Reply:
do we also need removal of secrets? (they are created by users, not the experiment itself, so should be fine, right? )

Reply:
Yes, I will add secrets also